erpnext/stock/doctype/delivery_note/delivery_note.py

```diff
@@ -119,6 +119,12 @@ def validate(self):
 		self.update_current_stock()
 
 		if not self.installation_status: self.installation_status = 'Not Installed'
+	
+	def validate_qty(self):
+		data = frappe.get_all("Delivery Note Item", filters={"parent" : self.name}, fields=["qty"])
```

Comment:
The `self` object already has the items data, so you don't need the `get_all`. Simply do:

```python
for item in self.items:
	if not item.qty:
		frappe.throw("Item quantity can not be zero")
```

Also, shouldn't this also be called in the `validate` method?